### PR TITLE
Feature/mutli user

### PR DESCRIPTION
**Pull request by pringelmann**

### CODE REVIEW

File: interfaces/slack.js [hunk 1]
```diff
@@ -195,23 +195,29 @@ const slackInterface = {
...
+                                              update.user
+                                          }: `,
+                                          err
+                                      )
+                                    : console.log('Message sent: ', res);
+                                update.editStatus('ts', res.ts);
+                                done();
+                            }
+                        ),
+                    1000 * i
```
Comment (jwfwessels) [inline interfaces/slack.js:218]: bit arb but ok 👍

File: server.js [hunk 2]
```diff
@@ -59,9 +59,68 @@ function commandRouter({ user_id, user_name, text }, response) {
...
+        case 'add':
+            let newChannel = text.split(' ')[1];
+            if (!newChannel) {
+                response.end(
+                    'You forgot to send me the channel name you would like to register for. (EG `general`)'
+                );
+                break;
+            }
+            if (newChannel.includes('<')) {
+                newChannel = newChannel.split('|')[1].replace('>', '');
```
Comment (jwfwessels) [inline server.js:71]: whats this for?

File: server.js [hunk 3]
```diff
@@ -59,9 +59,68 @@ function commandRouter({ user_id, user_name, text }, response) {
...
+                deleteChannel = deleteChannel.split('|')[1].replace('>', '');
+            }
+
+            firebase
+                .removeChannel(user_id, deleteChannel)
+                .catch(err => {
+                    response.end(err);
+                })
+                .then(() => {
+                    response.end("Don't let the door hit you on your way out.");
```
Comment (jwfwessels) [inline server.js]: maybe add  the channel name in italics 😆 
_leaving #channel_ ...

File: server.js (hunk 2 above)
Comment (pringelmann) [reply server.js:71]: If you tab autocomplete a channel name in slack it adds some strange formatting. Something like `<channel|engineering>`

File: interfaces/slack.js (hunk 1 above)
Comment (pringelmann) [reply interfaces/slack.js:218]: It's not arb - slack rate limiting is 1 msg/sec :stuck_out_tongue:

File: server.js [hunk 4]
```diff
@@ -59,9 +59,68 @@ function commandRouter({ user_id, user_name, text }, response) {
             firebase.deleteUser(user_id);
             response.end(":'( Goodbye " + user_name + ". It's been real.");
             break;
+        case 'add':
```
Comment (jwfwessels) [inline server.js:62]: I assume one doesn't write the # in front of the channel name cause its reserved?

Comment (pringelmann) [reply server.js:62]: you can do either or

File: interfaces/slack.js (hunk 1 above)
Comment (jwfwessels) [reply interfaces/slack.js:218]: oooh ok per user or per bot? how do you handle multiple ppl then?

File: server.js (hunk 2 above)
Comment (jwfwessels) [reply server.js:71]: interesting

File: interfaces/slack.js (hunk 1 above)
Comment (pringelmann) [reply interfaces/slack.js:218]: That gets managed by redis :)

**Pull request merged**